lib/matplotlib/contour.py

```diff
@@ -943,6 +943,16 @@ def __init__(self, ax, *args,
     alpha = property(lambda self: self.get_alpha())
     linestyles = property(lambda self: self._orig_linestyles)
 
+    @_api.deprecated("3.8", alternative="set_antialiased or get_antialiased",
+                     addendum="Note that get_antialiased returns an array.")
+    @property
+    def antialiased(self):
+        return all(self.get_antialiased())
+
+    @antialiased.setter
```

Comment:
deprecation needs to be added to the getter and setter individually (otherwise users who use only the setter will not be warned)

Reply:
This works for both - I've updated the test to show that.  I also tried adding the deprecation to both but with different warning messages: this gave me both messages whether I was setting or getting.